apps/system/js/nfc_handover_manager.js

```diff
@@ -525,26 +526,26 @@
       /*
        * Bug 979427:
        * After pairing we connect to the remote device. The only thing we
-       * know here is the MAC address, but the defaultAdapter.connect()
+       * know here is the MAC address, but the adapter.connect()
        * requires a BluetoothDevice argument. So we use getPairedDevices()
        * to map the MAC to a BluetoothDevice instance.
        */
-      var req = this.defaultAdapter.getPairedDevices();
-      var self = this;
-      req.onsuccess = function() {
+      var adapter = Service.query('Bluetooth.getAdapter');
+      var req = adapter.getPairedDevices();
+      req.onsuccess = () => {
```

Comment:
The return type of API is changed in v2. It's no longer return Dom request. It will return array in sync call so that above code won't be worked! Please find out solution here.

https://wiki.mozilla.org/B2G/Bluetooth/WebBluetooth-v2/BluetoothAdapter#getPairedDevices.28.29